src/main/java/edu/harvard/iq/dataverse/search/SearchIncludeFragment.java

```diff
@@ -452,7 +452,7 @@ public void search(boolean onlyDataRelatedToMe) {
             
             dataversePage.setQuery(query);
             dataversePage.setFacetCategoryList(facetCategoryList);
-            dataversePage.setFilterQueries(filterQueries);
+            dataversePage.setFilterQueries(filterQueriesFinal);
```

Comment:
@sekmiller what's the difference between filterQueries and filterQueriesFinal?

Reply:
It adds the "subtree" to the other filter queries, that is the dataverse that you're searching from. In line 349 of the SearchIncludeFragment filterQueriesFinal is what gets passed to the searchService for the Solr Search.

Reply:
Got it. all makes sense, though I think there could be some better naming here.